relay/utils/get_current_network_config.ts

```diff
@@ -19,9 +19,19 @@ const defaultConfig: Config = {
   DOMAIN_SYNC_COMMITTEE: '',
   GENESIS_VALIDATORS_ROOT: '',
 };
+export enum NetworkConfig {
+  Pratter = 'pratter',
+  Mainnet = 'mainnet',
+  Sepolia = 'sepolia',
+  Chiado = 'chiado',
+}
+
+export function isSupportedFollowNetwork(network: string): boolean {
+  return Object.values(NetworkConfig).includes(network as NetworkConfig);
+}
```

Comment:
Can you remind me why Enum but not union type?

Not a blocker, just wandering.

Reply:
Enum is what I had in mind, remade it with union as it makes more sence